doc/howto/json.md

```diff
@@ -110,20 +115,44 @@ Read the JSON data:
 ```{code-cell} ipython3
 %%sql
 SELECT *
-FROM read_json_objects('people.jsonl')
+FROM read_json_auto('people.json')
 ```
 
 ## Extract fields
 
-Extract fields from each JSON record:
+Extract fields from a JSON record:
+
+```{code-cell} ipython3
+%%sql
+SELECT 
+    name, 
+    friends[1] AS first_friend, 
```

Comment:
How are these 2 queries return the same result?

```
%%sql
SELECT
    json ->> '$.name' AS name,
    json ->> '$.friends[0]' AS first_friend, <---- index is 0, returns: Jake, John, John Kelly
    json ->> '$.likes.pizza' AS likes_pizza,
    json ->> '$.likes.tacos' AS likes_tacos
FROM read_json_objects('people.jsonl')
```

```
%%sql
SELECT 
    name, 
    friends[1] AS first_friend, 
    likes.pizza AS likes_pizza, <---- index is 1, returns: Jake, John, John Kelly
    likes.tacos AS likes_tacos
FROM read_json_auto('people.jsonl')
```

Reply:
yeah, I encountered the same while testing. I think it's a duckdb bug cause the docs say both are zero-based. so we'll leave it like this until they fix it

Reply:
@edublancas are you able to raise an issue for it?

Reply:
sure, will do it now!